docs/remote/devcontainerjson-reference.md

```diff
@@ -95,12 +95,12 @@ Variables can be referenced in certain string values in `devcontainer.json` in t
 
 | Variable | Properties | Description |
 |----------|---------|----------------------|
-| `${localEnv:VARIABLE_NAME}` | Any | Value of an environment variable on your local machine (in this case, called `VARIABLE_NAME`). Unset variables are left blank. To for example, this would set a variable to your local home folder on Linux / macOS or the user folder on Windows:<br/> `"remoteEnv": { "LOCAL_USER_PATH": "${localEnv:HOME}${localEnv:USERPROFILE}" }` |
+| `${localEnv:VARIABLE_NAME}` | Any | Value of an environment variable on your local machine (in this case, called `VARIABLE_NAME`). Unset variables are left blank. To for example, this would set a variable to your local home folder on Linux / macOS or the user folder on Windows:<br/> `"remoteEnv": { "LOCAL_USER_PATH": "${localEnv:HOME}${localEnv:USERPROFILE}" }` <br /><br /> ⚠️ For Codespaces, the local environment is in the cloud.|
```

Comment:
```suggestion
| `${localEnv:VARIABLE_NAME}` | Any | Value of an environment variable on your local machine (in this case, called `VARIABLE_NAME`). Unset variables are left blank. For example, this would set a variable to your local home folder on Linux / macOS or the user folder on Windows:<br/> `"remoteEnv": { "LOCAL_USER_PATH": "${localEnv:HOME}${localEnv:USERPROFILE}" }` <br /><br /> ⚠️ For Codespaces, the local environment is in the cloud.|
```

Reply:
Now that I see we have to handle the "local machine" case in a few spots, I think how you handle it here and in the area I commented above makes sense.

Reply:
Switched this to reference a host as well since the host can be anywhere.